src/frontend/next/src/components/AboutFooter.tsx

```diff
@@ -1,9 +1,10 @@
 import { makeStyles, useTheme } from '@material-ui/core/styles';
 import { Box, Divider, Grid, Typography, useMediaQuery } from '@material-ui/core';
-import GitHubIcon from '@material-ui/icons/Github';
+import GitHubIcon from '@material-ui/icons';
```

Comment:
this should be \@material-ui/icons/GitHub'
![image](https://user-images.githubusercontent.com/55034244/105049038-3cd30f00-5a3a-11eb-9ae1-183df18435d5.png)

Reply:
Fixed, thanks